neuralop/models/tests/test_fnogno.py

```diff
@@ -41,7 +47,10 @@ def test_fnogno(gno_transform_type, fno_n_modes):
     out = model(in_p, out_p, f, ada_in)
 
     # Check output size
-    assert list(out.shape) == [100, out_channels]
+    if gno_batched:
+        assert list(out.shape) == [batch_size, 100, out_channels]
+    else:
+        assert list(out.shape) == [100, out_channels]
 
     # Check backward pass
     loss = out.sum()
```

Comment:
We could also set the grad of `f` to True and do the gradient computation only on one sample from the output batch, i.e., `out[0]`. Then we can additionally check that the other samples in the input batch `f[1], ...` are not getting grads.